usefulUtilities/appLoader/README.md

```diff
@@ -0,0 +1,28 @@
+# App Loader
+When this script is attached to an entity as an entity script, the entity serves as an "app loader".
+
+## Features
+- App Loader will load scripts defined as hardcoded URLs in the JavaScript, AND will load scripts defined in the `userData` property of the attached entity.
+- When the user leaves the domain (or otherwise unloads the script attached to the entity), App Loader will stop client scripts that it loaded. App Loader will NOT stop client scripts that it did not load (i.e. those that were previously running).
+
+## Setup
+1. Add an entity to your domain.
+    - A Box primitive would work perfectly.
+    - Note that anyone who can modify the `userData` of this entity will be able to control which scripts other users load!
+    - A good place for this entity is right under the floor near your domain's spawn point. You want to ensure that all visitors to your domain load this entity.
+2. Add the below `userData` object to the attached entity's `userData`
+    1. Fill in the `apps` array in the `userData` by adding URLs to scripts that you want users to load.

```

Comment:
Did you mean to use 1. here?  It looks a little strange in the view file

Reply:
Indeed I did! Nested numbered lists render as (i) or (a) (i forget which one) in Markdown (at least on GitHub).

Reply:
got it, yeah it listed as (i).